diff -aurN terminatorX-3.82.orig/gnome-support/terminatorX.desktop terminatorX-3.82/gnome-support/terminatorX.desktop
--- terminatorX-3.82.orig/gnome-support/terminatorX.desktop	2004-03-26 20:13:24.000000000 +0100
+++ terminatorX-3.82/gnome-support/terminatorX.desktop	2005-04-13 21:06:26.000000000 +0200
@@ -1,10 +1,13 @@
 [Desktop Entry]
+Encoding=UTF-8
 Exec=terminatorX
 Name=terminatorX
 GenericName=Virtual turntable
 Comment=Versatile DJ scratching and mixing application.
+Comment[pl]=Wyrafinowany program do "skreczownia" oraz miksowania
 Terminal=false
 Type=Application
+MimeType=application/x-terminatorX
 Icon=terminatorX-app.png
-Categories=Application;AudioVideo;
+Categories=GTK;Audio;
 StartupNotify=true
