diff -aurN terminatorX-3.81.orig/gnome-support/Makefile.am terminatorX-3.81/gnome-support/Makefile.am
--- terminatorX-3.81.orig/gnome-support/Makefile.am	2002-11-24 22:40:00.000000000 +0100
+++ terminatorX-3.81/gnome-support/Makefile.am	2003-08-24 18:51:49.000000000 +0200
@@ -5,10 +5,8 @@
 	tx-gnome-uninstall\
 	terminatorX.desktop\
 	README.GNOME
-
-if GNOMEpresent
-gnomedir=@gnomedatadir@
-gnomeappdir=$(gnomedir)/gnome/apps/Multimedia
+	
+gnomeappdir=$(desktopdir)
 gnomeapp_DATA=terminatorX.desktop
 
 mimetypedir=$(gnomedir)/mime-info
@@ -16,7 +14,3 @@
 
 pixmapdir=$(gnomedir)/pixmaps
 pixmap_DATA=terminatorX-app.png terminatorX-mime.png
-
-endif
-
-	
